post_installation_configuration/coreos-layering.adoc

```diff
@@ -84,6 +84,28 @@ RUN rpm-ostree install libreswan && \
 Use the same base {op-system} image installed on the rest of your cluster. Use the `oc adm release info --image-for rhel-coreos-8` command to obtain the base image used in your cluster.
 ====
 
+[NOTE]
+====
+Some packages require additional configuration options, one common example is the kernel. When replacing the kernel you will need to enable rpm-ostree's cliwrap by running: 
+`rpm-ostree cliwrap install-to-root /`
+on your Containerfile.
+====
```

Comment:
Might I suggest some slight changes to the text? (Apologies if I don't have the proper wording.)
```
Some packages require additional configuration options. For example, if you are replacing the kernel, you need to enable rpm-ostree's cliwrap by running including the following command in your Containerfile: 
`rpm-ostree cliwrap install-to-root /`.
```

Reply:
@jmarrero 
Is `rpm-ostree cliwrap install-to-root` a common command? I did some web searching and couldn't find anything. I am a bit uncomfortable telling users that they must run specific non-OCP commands. Would this be better in a Knowledge Base article, perhaps we could link to it?
@cgwalters Any thoughts?

Reply:
I updated the sentence with the suggestions here.